fog/ingest/server/src/lib.rs

```diff
@@ -42,6 +42,7 @@ lazy_static::lazy_static! {
 // Helper to format a sequence as a comma-separated list
```

Comment:
This is dead code and should be removed, it got moved to https://github.com/mobilecoinfoundation/mobilecoin/blob/main/util/parse/src/lib.rs#L40